Sources/Brave/Frontend/Settings/SettingsViewController.swift

```diff
@@ -210,22 +210,42 @@ class SettingsViewController: TableViewController {
     return section
   }()
 
-  private lazy var featuresSection: Static.Section = {
+  @MainActor private func makeFeaturesSection() -> Static.Section {
```

Comment:
Same as above but feel free to double check it

Reply:
Done